deploy/008-configure-acm-commands-aggregator.ts

```diff
@@ -348,6 +379,20 @@ const revokePermissions: Permissions = {
     ...getRedstoneOracleRevokePermissions(ARBITRUMSEPOLIA_REDSTONE_ORACLE, ARBITRUMSEPOLIA_GUARDIAN),
     ...getOmniChainExecutorOwnerRevokePermissions(ARBITRUMSEPOLIA_OMNICHAIN_EXECUTOR_OWNER, ARBITRUMSEPOLIA_GUARDIAN),
   ],
+  basesepolia: [
+    ...getPrimeRevokePermissions(BASESEPOLIA_PRIME, BASESEPOLIA_GUARDIAN),
+    ...getPrimeLiquidityProviderRevokePermissions(BASESEPOLIA_PLP, BASESEPOLIA_GUARDIAN),
+    ...getResilientOracleRevokePermissions(BASESEPOLIA_RESILIENT_ORACLE, BASESEPOLIA_GUARDIAN),
+    ...getBoundValidatorRevokePermissions(BASESEPOLIA_BOUND_VALIDATOR, BASESEPOLIA_GUARDIAN),
+    ...getXVSVaultRevokePermissions(BASESEPOLIA_XVS_VAULT_PROXY, BASESEPOLIA_GUARDIAN),
+    ...getRewardDistributorRevokePermissions(BASESEPOLIA_GUARDIAN, true),
+    ...getIRMRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getPoolRegistryRevokePermissionsForWildcard(BASESEPOLIA_GUARDIAN),
+    ...getComptrollerRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getVTokenRevokePermissions(BASESEPOLIA_GUARDIAN),
+    ...getOmniChainExecutorOwnerRevokePermissions(BASESEPOLIA_OMNICHAIN_EXECUTOR_OWNER, BASESEPOLIA_GUARDIAN),
```

Comment:
I believe we are expected to revoke these permissions from the guardian once multichain governance is fully assured.

Reply:
Those permissions were also revoked on `opbnbtestnet`, `sepolia`, `arbitrumsepolia`, with the rest of the revokes. But they were not revoked for `zksyncsepolia` or `opsepolia`. Those permissions were not revoked in any mainnet yet.

So, I would not revoke those permissions on `basesepolia` yet (I would remove that line from the configuration). And I would revoke those permissions in the future.

```suggestion
```

Reply:
We have revoked those permissions on `opbnbtestnet,` `sepolia`,` arbitrumsepolia`  just to ensure compatibility and alignment with mainnet permissions. But we don't need them anymore as, we already have similar permissions set for mainnet and testnet on the networks like `zksync` and `opBNB` testnets.